api/envoy/extensions/filters/http/ratelimit/v3/rate_limit.proto

```diff
@@ -99,6 +98,10 @@ message RateLimit {
   // Disabled by default.
   XRateLimitHeadersRFCVersion enable_x_ratelimit_headers = 8
       [(validate.rules).enum = {defined_only: true}];
+
+  // Disables emitting the :ref:`x-envoy-ratelimited<config_http_filters_router_x-envoy-ratelimited>` header
+  // in case of rate limiting (i.e. 429 responses).
+  bool disable_x_envoy_ratelimited_header = 9;
```

Comment:
Disabling has the consequence of potentially making the request retriable. This is said in the link above, but I wonder if it is worth highlighting here, wdyt?

Reply:
Sure, why not.